bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -293,7 +293,8 @@ static Set<LedgerFragment> splitIntoSubFragments(LedgerHandle lh,
             assert false;
         }
 
-        long numberOfEntriesToReplicate = (lastEntryId - firstEntryId) + 1;
+        long numberOfEntriesToReplicate = firstEntryId == INVALID_ENTRY_ID && lastEntryId == INVALID_ENTRY_ID ? 0 :
```

Comment:
We can simplify the condition to `long numberOfEntriesToReplicate = firstEntryId == INVALID_ENTRY_ID ? 0 : (lastEntryId - firstEntryId) + 1;`

Reply:
done, PTAL

Reply:
LGTM